docs/pages/database-access/reference/cli.mdx

```diff
@@ -157,6 +157,66 @@ $ tctl auth sign --format=db --host=host1,localhost,127.0.0.1 --out=db --ttl=219
 | `--out` | Name prefix for output files. |
 | `--ttl` | Certificate validity period. |
 
+<Details title="Setting up RBAC for signing database certificates">
+
+The `tctl` user must have permissions to impersonate the Teleport Database
+Service user, `Db`, in order to generate a signed database certificate. To add
```

Comment:
```suggestion
Service role, `Db`, in order to generate a signed database certificate. To add
```

Technically "System role" is the correct term, but I think it's fine to say role. `Db` is definitely not a "user".